chainstate/test-suite/src/tests/helpers/pos.rs

```diff
@@ -113,7 +113,7 @@ pub fn create_stake_pool_data(
         anyonecanspend_address(),
         vrf_pk,
         destination,
-        0,
+        1000,
```

Comment:
Doesn't look like good randomization of test data.

Reply:
I believe it will just complecate tests. In all the use cases we either don't care or want 100% to the owner

Reply:
In that case, please call the function "create_stake_pool_data_with_all_reward_to_owner".